Mathlib/Algebra/Homology/HomotopyCategory/MappingCone.lean

```diff
@@ -541,6 +541,93 @@ lemma lift_desc_f {K L : CochainComplex C ℤ} (α : Cocycle K F 1) (β : Cochai
     liftCochain_v_descCochain_v φ α.1 β α' (Cochain.ofHom β') (zero_add 1) (neg_add_self 1) 0
     (add_zero 0) n n n (add_zero n) (add_zero n) n' hnn', Cochain.ofHom_v]
 
+
+section
+
+open Preadditive Category
+
+variable (H : C ⥤ D) [H.Additive]
+  [HasHomotopyCofiber ((H.mapHomologicalComplex (ComplexShape.up ℤ)).map φ)]
+
+/-- If `H : C ⥤ D` is an additive functor and `φ` is a morphism of cochain complexes
```

Comment:
Could the doc string (or a simple comment) explain why this is here separately from the unprimed version, and presumably direct the reader's attention to that?